signModule.js

```diff
@@ -0,0 +1,119 @@
+const defaultSignModule = {
+  submitButton: document.querySelector('#submit'),
+  emailInput: document.querySelector('#email'),
+  passwordInput: document.querySelector('#password'),
+  eyeIcons: document.querySelectorAll('img[alt=icon-eyes]'),
+  errorMapper: {
+    emailVoidError: "이메일을 입력해주세요.",
+    emailInvalidError: "올바른 이메일 주소가 아닙니다.",
+    passwordVoidError: "비밀번호를 입력해주세요.",
+    emailWrongError: "이메일을 확인해주세요.",
+    passwordWrongError: "비밀번호를 확인해주세요.",
+    passwordNotMatchError: "비밀번호가 일치하지 않습니다.",
+    passwordLengthError: "비밀번호는 8자 이상 20자 이하여야 합니다."
+  },
+  alertFunction: {
+    init: function (target) {
+      const initTargetString = target + " .alert"
+      const alertMessage = document.querySelectorAll(initTargetString);
+      if (alertMessage) {
+        alertMessage.forEach((item) => {
```

Comment:
item이라는 범용적인 변수보다는 message는 어떨까요?
```suggestion
        alertMessage.forEach((message) => {
```

Reply:
리팩토링에 적용하겠습니다.